meta/store.go

```diff
@@ -1664,6 +1678,27 @@ func (fsm *storeFSM) Apply(l *raft.Log) interface{} {
 	return err
 }
 
+func (fsm *storeFSM) applyRemovePeerCommand(cmd *internal.Command) interface{} {
+	ext, _ := proto.GetExtension(cmd, internal.E_RemovePeerCommand_Command)
+	v := ext.(*internal.RemovePeerCommand)
+
+	id := v.GetID()
+	addr := v.GetAddr()
+	//Remove that node from the peer
+	if id == fsm.id {
+		// if we are part of the raft cluster, remove us
+		if _, ok := fsm.raftState.(*localRaft); ok {
```

Comment:
Understood, but if I let it just execute on a remoteRaft, it will throw an error 100% of the time:

https://github.com/influxdb/influxdb/pull/4233/files#diff-88ae3f3a4cb80ed71213c7b103072d85R411

Would you prefer I return a nil (effectively a noop) and then this reads better?